src/Provider/ApiServiceProvider.php

```diff
@@ -65,7 +65,7 @@ public function register()
             'Dingo\Api\Console\Command\Docs',
         ]);
 
-        if (class_exists('Illuminate\Foundation\Application', false)) {
+        if (class_exists('Illuminate\Contracts\Foundation\Application', false)) {
```

Comment:
`laravel/lumen` would load `illuminate/contracts`, correct? And `Illuminate\Contracts\Foundation\Application` are under `illuminate/contracts`, correct?